lib/monitor.js

```diff
@@ -3,10 +3,12 @@
 var Events = require('events');
 var Http = require('http');
 var Https = require('https');
-var Os = require('os');
 var Hoek = require('hoek');
 var Items = require('items');
+var Os = require('os');
+var Readable = require('stream').Readable;
```

Comment:
In this case we wouldn't need to create a shortcut like this.  Look at https://github.com/hapijs/hapi/blob/master/lib/response.js#L541